third_party/move/move-compiler-v2/src/inliner.rs

```diff
@@ -989,10 +989,11 @@ impl<'env, 'rewriter> InlinedRewriter<'env, 'rewriter> {
 
     /// Convert a single-variable pattern into a `Pattern::Tuple` if needed.
```

Comment:
The comment no longer matches the functionality, I think?

Reply:
Fixed.